### DISCUSSION TITLE
[BUG]: updateShopReceipt Bug for was_shipped true

### DISCUSSION THREAD

**Discussion by mavofjune** [etsy/open-api#705] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

sending the value 'was_shipped' equals to true does not set the order as shipped. It appears to return no response as well. In the Etsy API V2, we did not have an issue. API v3 has this issue now

### Steps to reproduce

1. In our CodeIgnitor application, we submit the the curl request 
2. values include $postfields = 'was_shipped=true';
                $url_request = "https://openapi.etsy.com/v3/application/shops/". $store_id . "/receipts/" . $order_number;
3. we check for a response for is_shipped: true
4. No response was returned

### Expected behavior

Looking for response details for order:  is_shipped: true

**Comment by etsyachristensen** [comment]
We need to make sure we understand your issue. Are you trying to change the status on the receipt to "shipped"? Or are you wanting to get all receipts where the status of them is shipped?

**Comment by mavofjune** [comment]
we are trying to change the status of a single order to 'shipped'

**Comment by etsyachristensen** [comment]
We will be closing down the Issues list in favor of the Discussion board so we are converting this ticket to a discussion. It has been registered as an issue in our system to address.

**Comment by icemansparks** [comment]
Hi, this still seems to be an issue two years later - this is affecting seller rating... do you have any update on the status of the discussion?
